### ISSUE TITLE
No component factory found for PopoverContainerComponent.

### ISSUE THREAD

**Issue by touren** [isaacplmann/ngx-tour#80]
I checkout the master branch, and start a production app, i.e. ng serve --prod
Ngx-Tour Ngx-Bootstrap doesn't work.
Error: No component factory found for PopoverContainerComponent.

**Comment by isaacplmann** [comment]
I can't reproduce this, but #78 looks like it's the same bug.  Can you write down the exact steps you took?  What version of `ngx-bootstrap` is ending up in your `node_modules`?

**Comment by touren** [comment]
node v8.6.0
npm 5.6.0

Change package.json, 
start: ng serve ==> start: ng start --prod

```
npm install
npm run build:lib:all
npm start

@angular/compiler-cli@5.0.3 requires typescript@'>=2.4.2 <2.5.0' but 2.6.2 was found instead.
Using this version can result in undefined behaviour and difficult to debug problems.

Please run the following command to install a compatible version of TypeScript.

    npm install typescript@'>=2.4.2 <2.5.0'

To disable this warning run "ng set warnings.typescriptMismatch=false".

****************************************************************************************
This is a simple server for use in testing or debugging Angular applications locally.
It hasn't been reviewed for security issues.

DON'T USE IT FOR PRODUCTION!
****************************************************************************************
** NG Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
Date: 2018-01-30T20:40:29.124Z                                                          
Hash: 7718c6afd31ac47f4350
Time: 78388ms
chunk {0} 0.940fd6db7abd27acb72f.chunk.js (common) 28.3 kB  [rendered]
chunk {1} 1.2e52edf72e97d04000a0.chunk.js () 115 kB  [rendered]
chunk {2} 2.ccdcb650b6b910b5c7ea.chunk.js () 67.8 kB  [rendered]
chunk {3} 3.109f2cacc85e52ceb323.chunk.js () 142 kB  [rendered]
chunk {4} 4.9369df03c6dd534cd7bb.chunk.js () 67.5 kB  [rendered]
chunk {5} polyfills.2e35d91c97795a561cf4.bundle.js (polyfills) 65.6 kB [initial] [rendered]
chunk {6} main.f9cc8335d4da4c0a12bc.bundle.js (main) 479 kB [initial] [rendered]
chunk {7} styles.5cf98968a6d57e778c48.bundle.css (styles) 43.8 kB [initial] [rendered]
chunk {8} inline.febfa59b6f8433820f1c.bundle.js (inline) 1.57 kB [entry] [rendered]

webpack: Compiled successfully.
```

node_modules /ngx-bootstrap/package.json
`"version": "2.0.0-beta.9"`

-------------------------------------
Some Warnings from npm:
```
npm WARN ngx-tour-ngx-bootstrap@2.0.0-beta.9 requires a peer of ngx-bootstrap@^2.0.0-beta.9 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@2.0.0 requires a peer of @angular/common@^5.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@2.0.0 requires a peer of @angular/core@^5.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@2.0.0 requires a peer of @angular/router@^5.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@2.0.0 requires a peer of core-js@^2.5.1 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@2.0.0 requires a peer of rxjs@^5.5.2 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@2.0.0 requires a peer of zone.js@^0.8.18 but none is installed. You must install peer dependencies yourself.
```

**Comment by isaacplmann** [comment]
Thanks for the help reproducing it.

This should be fixed in `ngx-tour-ngx-bootstrap@2.0.0`.

**Comment by touren** [comment]
Thanks Issac, it's fixed.
